src/effects.js

```diff
@@ -333,7 +333,10 @@ function Animation( elem, properties, options ) {
 		animation = deferred.promise({
 			elem: elem,
 			props: jQuery.extend( {}, properties ),
-			opts: jQuery.extend( true, { specialEasing: {} }, options ),
+			opts: jQuery.extend( true, {
+				specialEasing: {},
+				easing: jQuery.easing._default
+			}, options ),
```

Comment:
I write a resume of the situation in another message.
But I like the way we put the default value **and after** the user rewrite them with the `.extend`.
I think (IMHO) we should do the same thing for the `duration` value.
And also for the `complete` callback with `jQuery.noop` maybe (it's only a suggestion).